config/jobs/kubernetes/kops/build_jobs.py

```diff
@@ -1171,6 +1171,17 @@ def generate_presubmits_e2e():
             extra_flags=["--dns=none"],
         ),
 
+        presubmit_test(
+            name="pull-kops-e2e-aws-nlb",
+            cloud="aws",
+            distro="u2204",
+            networking="calico",
+            extra_flags=[
+                "--api-loadbalancer-type=public"
```

Comment:
Missing trailing comma.

Reply:
Fixed in https://github.com/kubernetes/test-infra/pull/27839